### PR TITLE
[feat][minor] OSS benchmark - pick the model via args

### PR DESCRIPTION
**Pull request by blefaudeux**
# Before submitting

- [ ] Was this discussed/approved via a Github issue? (no need for typos, doc improvements)
- [x] Did you read the [contributor guideline](https://github.com/facebookresearch/fairscale/blob/master/CONTRIBUTING.md)?
- [ ] Did you make sure to update the docs?
- [ ] Did you write any new necessary tests?

## What does this PR do?
Helpful in debugging a bug that I have right now on another branch, and could be practical for some folks who want an easy way to benchmark OSS across typical vision workloads. It would be nice to add NLP workloads in there also, as a follow up

## PR review
Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

File: benchmarks/oss.py [hunk 1]
```diff
@@ -30,9 +30,12 @@ def dist_init(rank, world_size, backend):
     dist.init_process_group(backend=backend, init_method="tcp://localhost:29501", rank=rank, world_size=world_size)
 
 
-def get_problem(rank, data_size, batch_size, device):
-    # Standard RN101
-    model = resnet101(pretrained=False, progress=True).to(device)
+def get_problem(rank, data_size, batch_size, device, resnet: str):
+    # Select the desired model on the fly
+    print(f"Using {resnet} for benchmarking")
+    model = {"resnet18": resnet18, "resnet50": resnet50, "resnet101": resnet101, "resnet152": resnet152}[resnet](
```
Comment (min-xu-ai) [inline benchmarks/oss.py:36]: should we use importlib to import models directly from `torchvision.models`, that would avoid some glue code?

**Review by min-xu-ai — COMMENTED** [review]
one suggestion.

File: benchmarks/oss.py (hunk 1 above)
Comment (blefaudeux) [reply benchmarks/oss.py:36]: ahh I didn't know that, would you have a pointer ?

Comment (min-xu-ai) [reply benchmarks/oss.py:36]: an example:

```
import importlib


m1 = importlib.import_module('example.submodule')
print(m1)

m2 = importlib.import_module('.submodule', package='example')
print(m2)

print(m1 is m2)

```

you can turn string into import modules I think.

Comment (blefaudeux) [reply benchmarks/oss.py:36]: the models are not modules really, so it does not work OOB, but looking into the syntax

**Review by min-xu-ai — APPROVED** [review]
LGTM!

**Pull request merged**